pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, python2Packages, python2 }:
```

Comment:
`python2Packages` or `python2`. The interpreter is part of the package set, `python2Packages.python`, and the packages part of the interpreter, `python2.pkgs`

Reply:
Fixed